book/src/runtime.md

```diff
@@ -70,14 +81,19 @@ encodes.
 instructions to create memory before passing it to another program.  This
 instruction can be composed into a single transaction with the call to the
 program itself.
+
 2. Runtime guarantees that when memory is assigned to the program it is zero
 initialized.
+
 3. Runtime guarantees that a program's code is the only thing that can modify
 memory that its assigned to
+
 4. Runtime guarantees that the program can only spend tokens that are in
 accounts that are assigned to it
+
 5. Runtime guarantees the balances belonging to accounts are balanced before
 and after the transaction
+
 6. Runtime guarantees that multiple instructions all executed successfully when
```

Comment:
"multiple"?  Or "all" instructions are successfully executed in order for a transaction to be commited.